includes/class-wc-gateway-amazon-payments-advanced.php

```diff
@@ -2301,11 +2301,11 @@ public function filter_customer_field( $value ) {
 	 * Maybe hide standard WC checkout button on the cart, if enabled
 	 */
 	public function maybe_hide_standard_checkout_button() {
-		if ( ! $this->is_available() ) {
+		if ( ! $this->is_available() || $this->has_other_gateways_enabled() ) {
```

Comment:
Why this change? Seems unrelated to the settings.

Reply:
yes, I mixed up on the cherry pick and rebase. Let me fix it and force push both open prs